src/app/components/pages/Registration.tsx

```diff
@@ -252,7 +252,7 @@ const RegistrationPageComponent = ({user, updateCurrentUser, errorMessage, userE
                                             checked={confirmedOverThirteen}
                                             required
                                             label="I am at least 13 years old"
-                                            disabled={registrationUser.dateOfBirth !== null}
+                                            disabled={registrationUser.dateOfBirth !== undefined}
```

Comment:
DateInput sets the value to null on clear so to re-enable the checkbox after the input box is cleared we must also not disable when the date of birth is null.
```suggestion
                                            disabled={registrationUser.dateOfBirth}
```

Reply:
Implemented with comment about future restructuring of DoB handling